pkg/util/yaml.go

```diff
@@ -32,14 +33,20 @@ func UnmarshalYAML(filename string, object interface{}) error {
 	return nil
 }
 
-func Print(obj interface{}) error {
+func Print(obj interface{}, filename string) error {
```

Comment:
Suggest make new function Fprint(iowriter, interface) which can be used print Print() by passing stdout to Fprint.
So Print uses Fprint(stdout,..)

Reply:
Done